notebooks/spark/AerospikeSparkScala.ipynb

```diff
@@ -773,147 +774,6 @@
     "//Primary keys are not stored in bins(by default), hence only filters corresponding to bins are honored.  "
    ]
   },
-  {
```

Comment:
I am just curious why the "batchget query using __digest" section was removed from the AerospikeSparkScala notebook?  It seems to be still in the Python notebook. Thanks.

Reply:
spark 3.1.0 have all libraries shaded including java client. This implies, Key() function to create key for Aerospike primary key is not available outside hence we can't create digest. Thus I have to remove this example.